lib/super_good/solidus_taxjar/discount_calculator.rb

```diff
@@ -6,7 +6,7 @@ def initialize(line_item)
       end
 
       def discount
-        -line_item.promo_total
+        line_item.adjustments.select { |value| !value.tax? && value.eligible? }.sum(&:amount)
```

Comment:
```ruby
line_item.adjustments.non_tax.eligible.sum(:amount)
```

Reply:
Not going to implement this, as discussed in the weekly checkin, as we want this calculator to be able to handle in memory orders.